src/test/java/com/sampleweb/journey/CdVinylTests.java

```diff
@@ -0,0 +1,112 @@
+
+    private PageNavigator navigator = new PageNavigator();
+    private CdVinylPage cdVinylPage;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        cdVinylPage = navigator.navigateToPage(driver, CdVinylPage.PATH, CdVinylPage.class);
+    }
+
+    @Test(groups = "regression")
+    public void navigateToPage() {
+        //CDs & Vinyl dropdown
+        String dropDownPath = "//select[@class=\"nav-search-dropdown searchSelect\"]/option[@value=\"search-alias=popular\"]";
+
+        driver.findElementByXPath(dropDownPath).click();
+
+        cdVinylPage.setSearchCriteria("").clickSubmitButton();
+
+        assertThat(cdVinylPage.isLoaded(), is(true));
+    }
+
+
+    @Test(groups = "regression")
+    public void titleTest() {
+        Assert.assertTrue( driver.getTitle().contains(cdVinylPage.TITLE) );
+    }
+
+    @Test(groups = "regression")
+    public void checkGlobalStoreSection() {
+        WebElement storeElement = driver.findElementByXPath(cdVinylPage.GLOBAL_STORE_CHECKBOX_PATH);
+
+        //if query can't find an object, fail test
+        if(storeElement == null) {
+            Assert.fail();
+        }
+    }
+
+    @Test(groups = "regression")
+    public void checkAverageCustomerReview() {
+        WebElement reviewElementTitle = driver.findElementByXPath("//h4[text() = \"Avg. Customer Review\"]");
+
+        if(reviewElementTitle == null) {
+            Assert.fail();
+        }
+
+        //Only stars are in a unordered list with icons
+        WebElement starElementList = driver.findElementByXPath("//ul/div/li/span/a/i");
+        if(starElementList == null) {
+            Assert.fail();
+        }
+
+    }
+
+    //Its grouped under music so title is the 4th element
+    @Test(groups = "regression")
+    public void fourthLinkHeaderIsCategory() {
+        String elementText = driver.findElementByXPath(cdVinylPage.CD_VINYL_LINK_PATH).getText();
+        Assert.assertEquals(elementText, cdVinylPage.TITLE);
+    }
+
+    @Test(groups = "regression")
+    public void checkfourthLinkRedirect() {
+        driver.findElementByXPath(cdVinylPage.CD_VINYL_LINK_PATH).click();
+
+        //Same title means on the same page
+        titleTest();
+    }
+
+    @Test(groups = "regression")
+    public void checkPrimeCheckBox() {
+        WebElement primeCheckbox = driver.findElementByXPath(cdVinylPage.PRIME_CHECKBOX_PATH);
+
+        //if query can't find an object, fail test
+        if(primeCheckbox == null) {
+            Assert.fail();
+        }
+
+        primeCheckbox.click();
+
+        //Xpath goes through div and data-asin, according to amazon data-asin is a identification number
+        //https://www.nchannel.com/blog/amazon-asin-what-is-an-asin-number/ so each section should have one for each track
+        //Search for prime image.
+        //Only checks prime on first description ie Audio CD.
+        List<WebElement> primeCheckboxList = driver.findElementsByXPath(
```

Comment:
an easier xpath would be //div[@class="s-result-list s-search-results sg-row"]/*